dolt/constants.py

```diff
@@ -1,10 +1,10 @@
-# TODO: move these to settings?
+from django.conf import settings
 
 
-DB_NAME = "nautobot"
+DB_NAME = settings.DOLT_DB
 
-GLOBAL_DB = "global"
+GLOBAL_DB = settings.GLOBAL_DB
 
-DOLT_DEFAULT_BRANCH = "main"
+DOLT_DEFAULT_BRANCH = settings.DOLT_DEFAULT_BRANCH
```

Comment:
If possible we should use the plugin settings to define the default branch instead of a the global settings